jobs/mariadb-blacksmith-plans/templates/plans/standalone/manifest.yml

```diff
@@ -4,7 +4,7 @@ meta:
     azs: z1
   admin_password: (( vault $CREDENTIALS "/standalone/system:password" ))
   admin_username: (( vault $CREDENTIALS "/standalone/system:username" ))
-  credential_name_field: "service_instance_db"
+  credential_name_field: (( vault $CREDENTIALS "/standalone/system:dbname" ))
```

Comment:
What is a "credential name field"?

Reply:
A name that will be updated to something better

Reply:
updated